apps/reactStorybook/module-federation.config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  name: 'reactStorybook',
+  remotes: ['reactRemoteUI'],
```

Comment:
I'm curious on how the remote is getting resolved? 

Also can I customise the remotes to provide full URL of the remote??

Reply:
This is a normal NX MF configuration. I create a custom util to add the possibility to consume the remote app/component in the storybook. The NX utils `withModuleFederation` change the Webpack config to `{ type: " module" }` and I cannot find a way to consume it in the storybook. For this reason, I decided to create [my own](https://github.com/module-federation/universe/blob/ac0445627d1785fa1764728a91bbfd6d65713e53/packages/storybook-addon/src/utils/with-module-federation.ts) `withModuleFederation` util.
Remote is resolved in the util `withModuleFederation`.

[Here](https://nx.dev/recipes/module-federation/faster-builds#) you can find more details on how to configure MF for NX app

Reply:
what if users don't wanna use NX? I see that this plugin is tightly locked with NX.

Reply:
No, you can pass a normal MF config. You can see the readme. I need to update read me with example of nx config.